tests/embind/test_val.cpp

```diff
@@ -72,8 +72,34 @@ EMSCRIPTEN_BINDINGS(test_bindings)
   emscripten::function("makeDummy", &makeDummy, emscripten::allow_raw_pointers());
 }
 
+void __attribute__((noinline)) val_array_benchmark() {
```

Comment:
Not sure what to do about this - it's a bit weird to have a function that isn't called. Maybe @sbc100 has opinions.

Reply:
We don't have a good place for this kind of thing sadly.   We do have `test_benchmarks.py` but those are generally large whole programs used to benchmark emscripten against other runtimes (mostly native).

Out test code is already not the tidiest but thats not really a good reason for letting this stick around.

If it only takes 50ms why not just call it?  if the output comes before the first line of the expected `.out` text (which it looks like it does) then it will be ignored.

Reply:
There's an embind_benchmark.cpp and embind.behcmark.js file we could move the benchmarks too. Unfortunately, that file is not currently compiling.

Reply:
Benchmark file will be fixed in #17300.

Reply:
I'll move the benchmark to there.  

I once noticed the `embind_benchmark.cpp` before but it just can't run up.  So in #17186 I add a separate `val_builder_benchmark.cpp`, and it can compile successfully but need `-v' command-line option to see the output of printf.  I opened an issue #17282.  @brendandahl  Please a look.